storage/redis/peer_store.go

```diff
@@ -1,30 +1,36 @@
 // Package redis implements the storage interface for a Chihaya
-// BitTorrent tracker keeping peer data in redis with hash.
-// There two categories of hash:
+// BitTorrent tracker to keep peer data in redis.
 //
-// - IPv{4,6}_{L,S}_infohash
-//	To save peers that hold the infohash, used for fast searching,
-//  deleting, and timeout handling
+// In general, each swarm is tracked in two hashes per address family:
 //
-// - IPv{4,6}
-//  To save all the infohashes, used for garbage collection,
-//	metrics aggregation and leecher graduation
+// - swarm_<infohash>_IPv{4|6}_{seeders|leechers} -> <peer key> -> <last modified time>
```

Comment:
I guess.. I renamed them to be more verbose because it took me so long to figure out what was happening with the short names :)

Reply:
Ah, I found something better: Key names don't need to be ASCII or whatever, so I can use the 20-byte infohash instead of the 40-byte hex representation...

Reply:
I used the same trick with the infohash peer count keys. Btw, in `redis-cli` they are displayed like this now: `"swarm_\xb2\xbcP\\\xe9\xca\x92\r\x1a\xe6\xb7\x0e\xc5\xf8\xea A\x1ab\xbd_IPv4_leechers"`, so that works...